.github/workflows/.env

```diff
@@ -1,7 +1,10 @@
 # TODO - update Context of the account where the role vending machine should live (eg. IamAdmin account)
 AWS_ACCOUNT_ID=111111111111
-# Role from the bootstrap script
+TF_VAR_rvm_account_id=111111111111
```

Comment:
Double-documenting the account ID is dumb, let's just fix the workflow references so that they reference this TF_VAR_rvm_account_id instead